packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -37,10 +37,8 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
 
   const popperRef = React.useRef<PopperJS>()
   const contentRef = React.useRef<HTMLElement>(null)
-  const latestPlacement = React.useRef<PopperJS.Placement>(proposedPlacement)
-  const [computedPlacement, setComputedPlacement] = React.useState<PopperJS.Placement>(
-    proposedPlacement,
-  )
+  const latestPlacement = React.useRef<PopperJS.Placement>()
+  const [computedPlacement, setComputedPlacement] = React.useState<PopperJS.Placement>()
```

Comment:
Is it correct that we don't have initial value there more?

Reply:
looks like we don't need it, but I reverted for consistency, no harm there